schema/me/conversation.js

```diff
@@ -71,6 +72,7 @@ export const ConversationInitiatorType = new GraphQLObjectType({
       description: "The type of participant, e.g. Partner or User",
       type: new GraphQLNonNull(GraphQLString),
     },
+    initials: initials("name"),
```

Comment:
..but then I wasn't sure if we could access that easily from the conversation. Would this make more sense? (2/2)